config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -168,3 +168,13 @@ data:
       - "Quota '[A-Z_]*' exceeded"
       installFailingReason: GeneralQuotaExceeded
       installFailingMessage: Quota exceeded
+    - name: AWSSubnetDoesNotExist
```

Comment:
we should move these AWS search strings with the rest of pre-existing ones specific to AWS around line 84 in this file